src/components/network-area-diagram-viewer/nad-viewer-parameters.ts

```diff
@@ -102,6 +102,12 @@ export interface NadViewerParametersOptions {
 
     // Size in pixel of the margin that is added to hoverable objects to help the user stay over them.
     hoverPositionPrecision?: number | null;
+
+    // Whether enabling adaptive zoom
+    enableAdaptiveZoom?: boolean;
```

Comment:
It seems that the zoom is adaptive, which is quite unclear to me. A more precise comment might be enough, but maybe we should rename this to include the text keyword? As it's the text which is adaptive (depending on the zoom level)